Documentation/Blazorise.Docs/Pages/News/2024-10-15-release-notes-170.razor

```diff
@@ -312,4 +312,12 @@
     We have improved the way how our Fluent Flex utilities work. Previously it was not possible to define a breakpoint on per-flex rule, which made it hard to define different flex rules for different breakpoints. Now you can define the flex rules for each breakpoint separately, and you can also define the order of the flex rules.
 </Paragraph>
 
+<Heading Size="HeadingSize.Is3">
+    Disable Video Fullscreen
+</Heading>
+
+<Paragraph>
+    Added <Code>ClickToFullscreen</Code> parameter to the <Code Tag>Video</Code> component. This can be useful if you want to prevent the video from going fullscreen when the user double clicks on the video player.
```

Comment:
Seems like "Disable Video Fullscreen" is a byproduct of disabling the **ClickToFullscreen**.  So the title really should be "Double Click To FullScreen" or something along those lines? And then on the text we can inform, that's useful to disable it in order to prevent fullscreen.

Reply:
i.e: "You can now control whether double clicking the video player will enter full screen. By disabling this parameter, the user will not be able to enter full screen in this way."

(Is there a full-screen button or programatic way to enter fullscreen? If there is then we need to be explicit that this prevents full screen only on double click, but full screen can still be achieved through the button or programatically.)

Reply:
Yes, there is still an option to define a button by defining it as part of the `ControlsList` parameter.